### PR TITLE
Schedule Page

### PR DESCRIPTION
**Pull request by Jasem84**
adding the schedule page content

### CODE REVIEW

File: src/views/schedule.jade [hunk 1]
```diff
@@ -1,4 +1,100 @@
...
+			td Wisconsin Storm 
+			td.rangers
+				img(src = "/favicon-32x32.png")
+				span Minnesota Rangers  
+			td Sat. April 30th 
+			td 12:00 PM
+			td
+		tr
+			td Chicago Fury   
+			td.rangers Minnesota Rangers
```
Comment (jaboc83) [inline src/views/schedule.jade]: Need logo here.

File: src/views/schedule.jade [hunk 2]
```diff
@@ -1,4 +1,152 @@
...
+			th Away
+			th Date
+			th Time
+			th Result
+		tr
+			td.rangers
+				img(src = "/favicon-32x32.png")
+				span  Minnesota Rangers
+			td
+				img(src = "/broncos-32x32.png")
```
Comment (jaboc83) [inline src/views/schedule.jade]: Can you put the other team icons into a subfolder in images. images/otherLogos/  or something like that. I'd like to keep the root folder clean of non-ranger stuff if possible

Comment (Jasem84) [inline src/views/schedule.jade]: I wasn't quite sure what to do with those. I'll fix that this afternoon.

Comment (jaboc83) [inline src/views/schedule.jade]: thanks!

**Pull request merged**